palisade-service/src/unit-tests/java/uk/gov/gchq/palisade/service/palisade/service/PalisadeServiceTest.java

```diff
@@ -56,18 +56,29 @@ void setUp() {
     @Test
     void testRegisterDataRequest() {
         CompletableFuture<String> token = service.registerDataRequest(PALISADE_REQUEST);
-        assertThat(token.join()).isEqualTo(COMMON_UUID.toString());
-        assertThat(sinkCollection).usingRecursiveComparison().isEqualTo(List.of(new TokenRequestPair(token.join(), AUDITABLE_PALISADE_REQUEST)));
+
+        assertThat(token.join())
+                .as("Check the value of the returned Token")
+                .isEqualTo(COMMON_UUID.toString());
+        assertThat(sinkCollection).as("Recursively check the returned TokenRequestPair")
+                .usingRecursiveComparison()
+                .isEqualTo(List.of(new TokenRequestPair(token.join(), AUDITABLE_PALISADE_REQUEST)));
     }
 
     @Test
     void testErrorMessage() {
         CompletableFuture<String> token = service.registerDataRequest(PALISADE_REQUEST);
         service.errorMessage(PALISADE_REQUEST, token.join(), attributes, ERROR);
-        assertThat(token.join()).isEqualTo(COMMON_UUID.toString());
+        assertThat(token.join())
+                .as("Check the value of the returned Token")
+                .isEqualTo(COMMON_UUID.toString());
         assertAll(
-                () -> assertThat(sinkCollection.getLast().first()).isEqualTo(token.join()),
-                () -> assertThat(sinkCollection.getLast().second().getAuditErrorMessage()).usingRecursiveComparison().ignoringFields("timestamp")
+                () -> assertThat(sinkCollection.getLast().first())
+                        .as("Check the Token value from the error-topic")
+                        .isEqualTo(token.join()),
+                () -> assertThat(sinkCollection.getLast().second().getAuditErrorMessage())
+                        .as("Recursively check the value of the AuditErrorMessage")
+                        .usingRecursiveComparison().ignoringFields("timestamp")
                         .isEqualTo(List.of(new TokenRequestPair(token.join(), AUDITABLE_PALISADE_ERROR)).get(0).second().getAuditErrorMessage())
```

Comment:
add the top assertion to the groupAssertion

the AuditErrorMessage test can be better formatted
and if you're checking both objects individually, theres not much point in testing that they are in a TokenRequestPair seperatly
you can also add the timestamp assertion as mentioned earlier
```
assertAll(
                () -> assertThat(token.join())
                        .as("Check the value of the returned Token")
                        .isEqualTo(COMMON_UUID.toString()),

                () -> assertThat(sinkCollection.getLast().first())
                        .as("Check the Token value from the error-topic")
                        .isEqualTo(token.join()),

                () -> assertThat(sinkCollection.getLast().second().getAuditErrorMessage())
                        .as("Recursively check the value of the AuditErrorMessage")
                        .usingRecursiveComparison()
                        .ignoringFields("timestamp")
                        .isEqualTo(AUDITABLE_PALISADE_ERROR.getAuditErrorMessage())
        );
```

Reply:
Updated.